dfn_configs.py

```diff
@@ -268,4 +268,7 @@ def __init__(self,
               name_prefix='Sensirion ', lead_width=0.35, no_exp=False, pin1_corner_dx_dy=0.2),
     DfnConfig(3.0, 3.0, 1.0, 6, 1.1, 1.20, 0.4, 2.4, 1.5, 'sensirion,sht,sht2x,sht20,sht21,sht25',
               name_prefix='Sensirion ', lead_width=0.4, no_exp=False, pin1_corner_dx_dy=0.2),
+
+    # Microchip
+    DfnConfig(2.0, 3.0, 0.5, 8, 0.90, 1.00, 0.40, 1.75, 1.45, '', no_exp=False),
```

Comment:
Should we add the `Microchip ` prefix, similar to `Sensirion ` above?

Reply:
I didn't add a `Microchip ` prefix, because it is added to the Base library, and it is not considered a Microchip-specific device. As asked by Urban:

> What's the difference between "Microchip DFN50P200X300X90-8T175X145" and "Linear Technology DFN50P200X300X90-8T175X145"?

Reply:
Where are the Sensirion thingies added? Not to the base library then but to the Sensirion library? @dbrgn?

Reply:
Yes, afaik those are in the Sensirion library. They are specific to Sensirion due to the holes and stuff. This package here is a generic DFN-package with an exposed pad in non-standardized dimensions.

Reply:
Alright.